README.md

```diff
@@ -142,9 +140,6 @@ From reporting a bug to submitting a pull request: every contribution is appreci
 Report bugs, ask questions and request features using [Github issues][github-issues].
 If you want to contribute to the code of this project, please read the [Contribution Guidelines][contributing].
 
-[travis-status]: https://travis-ci.org/rosscdh/mkdocs-markdownextradata-plugin.svg?branch=master
-[travis-link]: https://travis-ci.org/rosscdh/mkdocs-markdownextradata-plugin
```

Comment:
please undo this

Reply:
I removed those because they are unused links and travis-ci.org is ceased.

Reply:
right you are! I hadn't even noticed!